check_image.py

```diff
@@ -16,22 +15,29 @@
 
 for i in images:
   image_split = i.split("/")
-  if len(image_split) == 3:
-    registry, repo, name = image_split
-  else:
+
+  if len(image_split) < 3:
     print("WARNING: Registry for image %s is not set, assuming docker.io. Cannot verify image existance there. Skipping." % i)
     continue
+
+  registry = image_split[0]
+  repo = '/'.join(image_split[1:-1])
```

Comment:
This is the most important line here, since it may have more than one path component, we are joining all the remaining components together.